doxygen/main.md

```diff
@@ -40,7 +40,7 @@ DMX receiver / RDM responder.
 
 ## Common {#main-features-common}
 
-- \ref usb-logging, with adjustable log levels.
+- \ref usb-logging, with adjustable log levels. Requires OS X 10.7 or greater.
```

Comment:
Can we word this in such a way it doesn't imply you need OS X. We should probably also clarify that Windows won't work without a driver file.